userpatches/customize-image.sh

```diff
@@ -246,6 +232,42 @@ uri=http://detectportal.firefox.com/canonical.html
 interval=300
 NMCONF
 
+# ============================================================================
+# Modify config.txt to put GPIOs into correct state for OpenWB
+# ============================================================================
+apt-get install -y --no-install-recommends labwc wayfire seatd xdg-user-dirs firefox-esr swayidle wlopm
+
+mkdir -p /home/admin/.config/labwc || true
+cat >/home/admin/.config/labwc/autostart <<'LABWCAUTOSTART'
+/usr/bin/firefox --kiosk http://localhost:7070/ &
```

Comment:
I'm connecting to evcc directly here (instead of doing https through caddy), because there is no need for https on localhost and to prevent the unknown certificate warning in firefox.